PackageDownloadsExample/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using NuGet.Configuration;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
+
+namespace PackageDownloadsExample
+{
+    /// <summary>
+    /// Find the download count for each version of a package.
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// Find the download count for each version of a package.
+        /// </summary>
+        /// <param name="packageId">The package ID. Example: "newtonsoft.json"</param>
+        public static async Task Main(string packageId)
+        {
+            if (string.IsNullOrEmpty(packageId))
+            {
+                Console.WriteLine("The --package-id option is required. Run --help for more information");
+                return;
+            }
+
+            // 1. Find the latest search endpoint using the NuGet Service Index API
+            // See: https://docs.microsoft.com/en-us/nuget/api/service-index
+            var source = new PackageSource("https://api.nuget.org/v3/index.json");
+            var providers = Repository.Provider.GetCoreV3();
+            var repository = new SourceRepository(source, providers);
+
+            var serviceIndex = await repository.GetResourceAsync<ServiceIndexResourceV3>();
```

Comment:
Can you use `RawSearchResourceV3` instead?

Reply:
Ah I didn't realize `RawSearchresponseV3` returned `JObject`s. I've updated the code sample, please let me know what you think

Reply:
Looks good.